### PR TITLE
Update cli and lib changelogs

### LINKED ISSUE #663: Update 2.2.0 changelog files
Make sure to port previous patches' changelogs

### PR DESCRIPTION
**Pull request by facuspagnuolo**
Fixes #663

### CODE REVIEW

File: packages/cli/changelog.md [hunk 1]
```diff
@@ -4,14 +4,83 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Next
+
+### Added
+- Contributing guidelines ([#553](https://github.com/zeppelinos/zos/pull/553))
+- Example using truffle migrations ([#416](https://github.com/zeppelinos/zos/pull/416))
+- Integration tests using `truffle-hdwallet-provider` ([#473](https://github.com/zeppelinos/zos/pull/473))
```
Comment (spalladino) [inline packages/cli/changelog.md]: This is not a new feature, but a fix to integration tests that already used it but were failing

File: packages/cli/changelog.md [hunk 2]
```diff
@@ -4,14 +4,83 @@ All notable changes to this project will be documented in this file.
...
+- Tests for `verify` command ([#468](https://github.com/zeppelinos/zos/pull/468))
+- Migration process for projects using previous versions of ZeppelinOS v2 ([#621](https://github.com/zeppelinos/zos/pull/621))
+- Support compiling with globally installed Truffle ([#597](https://github.com/zeppelinos/zos/pull/597))
+- Support receiving booleans via the CLI ([#623](https://github.com/zeppelinos/zos/pull/623))
+- Parse scientific notation numbers from the CLI ([#567](https://github.com/zeppelinos/zos/pull/567))
+- Linter pre-commit githook ([#620](https://github.com/zeppelinos/zos/pull/620))
+
+### Changed
+- Update Web3 to version `1.0.0-beta.37` ([#572](https://github.com/zeppelinos/zos/pull/572))
```
Comment (spalladino) [inline packages/cli/changelog.md]:
```suggestion
- **Update Web3 to version `1.0.0-beta.37`** ([#572](https://github.com/zeppelinos/zos/pull/572))
```

Comment (spalladino) [reply packages/cli/changelog.md]: Isn't this actually part of the lib changelog?

File: packages/cli/changelog.md [hunk 3]
```diff
@@ -4,14 +4,83 @@ All notable changes to this project will be documented in this file.
...
+- Add `lodash` functions separately ([#555](https://github.com/zeppelinos/zos/pull/555)) (thanks @slorenzo!)
+- Update Etherscan api key url ([#474](https://github.com/zeppelinos/zos/pull/474))
+- Polish `Truffle` util methods used in CLI ([#507](https://github.com/zeppelinos/zos/pull/507))
+- Avoid transferring proxies to `App` on publish ([#609](https://github.com/zeppelinos/zos/pull/609))
+
+### Fixed
+- Bytecode comparison in `SimpleProject` ([#580](https://github.com/zeppelinos/zos/pull/580))
```
Comment (spalladino) [inline packages/cli/changelog.md]: Should be in lib, as SimpleProject is not part of the CLI

File: packages/cli/changelog.md [hunk 4]
```diff
@@ -4,14 +4,83 @@ All notable changes to this project will be documented in this file.
...
 
+## Next
+
+### Added
+- Contributing guidelines ([#553](https://github.com/zeppelinos/zos/pull/553))
+- Example using truffle migrations ([#416](https://github.com/zeppelinos/zos/pull/416))
+- Integration tests using `truffle-hdwallet-provider` ([#473](https://github.com/zeppelinos/zos/pull/473))
+- Tests for `verify` command ([#468](https://github.com/zeppelinos/zos/pull/468))
+- Migration process for projects using previous versions of ZeppelinOS v2 ([#621](https://github.com/zeppelinos/zos/pull/621))
+- Support compiling with globally installed Truffle ([#597](https://github.com/zeppelinos/zos/pull/597))
```
Comment (spalladino) [inline packages/cli/changelog.md]: I'd remove this one if we are already listing it in 2.1.1

File: packages/lib/changelog.md [hunk 5]
```diff
@@ -4,9 +4,67 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Next
+
+### Added
+- Contributing guidelines ([#553](https://github.com/zeppelinos/zos/pull/553))
+- New `ProxyAdmin` contract to manage proxy instances ([#588](https://github.com/zeppelinos/zos/pull/588) and [#625](https://github.com/zeppelinos/zos/pull/625)) (see [audit report](https://medium.com/nomic-labs-blog/zeppelinos-audit-report-iii-602714cb92d4))
```
Comment (spalladino) [inline packages/lib/changelog.md]:
```suggestion
- **New `ProxyAdmin` contract to manage proxy instances** ([#588](https://github.com/zeppelinos/zos/pull/588) and [#625](https://github.com/zeppelinos/zos/pull/625)) (see [audit report](https://medium.com/nomic-labs-blog/zeppelinos-audit-report-iii-602714cb92d4))
```

File: packages/lib/changelog.md [hunk 6]
```diff
@@ -4,9 +4,67 @@ All notable changes to this project will be documented in this file.
...
+- Linter pre-commit githook ([#620](https://github.com/zeppelinos/zos/pull/620))
+
+### Changed
+- Update Web3 to version `1.0.0-beta.37` ([#572](https://github.com/zeppelinos/zos/pull/572))
```
Comment (spalladino) [inline packages/lib/changelog.md]:
```suggestion
- **Update Web3 to version `1.0.0-beta.37`** ([#572](https://github.com/zeppelinos/zos/pull/572))
```

File: packages/cli/changelog.md (hunk 2 above)
Comment (facuspagnuolo) [reply packages/cli/changelog.md]: I put it in both changelogs since cli is also depending on web3 1.x

**Pull request merged**